src/celo/parsers.ts

```diff
@@ -47,6 +50,11 @@ export function parseTransaction<TSerialized extends CeloTransactionSerialized>(
       serializedTransaction as TransactionSerializedCIP64,
     ) as ParseTransactionReturnType<TSerialized>
 
+  if (serializedType === '0x7a')
+    return parseTransactionCIP66(
+      serializedTransaction as TransactionSerializedCIP66,
+    ) as ParseTransactionReturnType<TSerialized>
```

Comment:
doesnt this need to go first? yeah. i think the test for 66 must go before the test for 64

Reply:
oh no i was thinking of serializer. nevermind